models/POP/README.rst

```diff
@@ -6,232 +6,344 @@ Contents
 ========
 
 #. `Overview`_
-#. `Observations`_
+
+#. `Development history`_
+
+   #. `LANL POP`_ 
+
+   #. `CESM POP2`_
+
+#. `Detailed instructions for using DART and CESM POP2 on NCAR's supercomputer`_ 
+
+   #. `Shell scripts`_
+
+   #. `Other files needed for assimilation`_
+
 #. `Namelist`_
-#. `Files`_
-#. `Terms of Use`_
+
+#. `Terms of use`_
+
+#. `References`_
 
 Overview
 ========
 
-The **Parallel Ocean Program (POP)** may be used with the **Data Assimilation
-Research Testbed (DART)**. Both main variants - `LANL POP
-<https://climatemodeling.science.energy.gov/projects/climate-ocean-and-sea-ice-modeling-cosim>`_
-and `CESM2.0 POP2 <https://www.cesm.ucar.edu/models/cesm2/ocean/>`_ have been
-tested under the Lanai framework, only use with **CESM** is supported under the 
-Manhattan framework.
+This document describes the DART interface to the Parallel Ocean Program (POP).
+It covers the `Development history`_ of the interface with two implementations
```

Comment:
````suggestion
It covers the `development history`_ of the interface with two implementations
````

Reply:
I gleaned the link text capitalization rules from the [Link text](https://developers.google.com/style/link-text) section of the Google developer style guide. In brief:

1. Section headers use sentence capitalization
2. Mid-sentence link text copies section headers

The end result is that there's a capitalization mid-sentence, which seems a little strange. But the style guide claims that this linking structure minimizes link fatigue. I'm not tied to this structure so if we don't want to follow it, that's fine by me. I just wanted to explain why I wrote links this way.

Reply:
oh I totally read the link text section as  "capialize as part of a sentence". 
I'll get rid of the suggestions.